lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,109 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+
+        # Native options
+        ignore: one_or_more_strings?,
+      )
+
+      let :issue, object(
+        shellcheck: object?(id: string, severity: string, line: integer, column: integer, message: string, url: string),
```

Comment:
[suggestion] Does it make sense to have also the top-level `severity` like other analyzers?

```suggestion
        shellcheck: object?(id: string, severity: string, line: integer, column: integer, message: string, url: string),
        severity: string?,
```

Reply:
I don't think so. Issues other than those reported by `shellcheck` (under actionlint) do not have severity information attached. We would have to categorize them by ourselves.

Reply:
I see. Let's keep it as-is. 👍🏼 

FYI, users can filter by severities on the UI:

<img width="216" alt="image" src="https://user-images.githubusercontent.com/473530/133541700-c766f16e-e796-49ab-8463-a58606186bf4.png">